packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -406,6 +410,13 @@ class CupertinoDatePicker extends StatefulWidget {
   /// Whether to to show day of week alongside day. Defaults to false.
   final bool showDayOfWeek;
 
+  /// The uniform height of all children.
+  ///
+  /// All children will be given the [BoxConstraints] to match this exact
+  /// height. Must not be null and must be positive.
+  /// Default to a value that matches the default iOS date picker wheel.
+  final double itemExtent;
```

Comment:
If this is always the height, then why not call it `itemHeight`?

Reply:
I call it `itemExtent` because, `CupertinoPicker.builder`  has a property `itemExtent` to which the actual `itemExtent` is passed. 

Should I rename `itemExtent` to `itemHeight` for both `CupertinoDatePicker` and `CupertinoPicker` ?

Reply:
If it weren't going to break things, I'd say yes, but I guess since it already exists, let's be consistent.  It's not that bad a name, but height would have been clearer.